tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -134,59 +134,79 @@ public function testFixProvider()
                 '<?php function xyz(
                     $a=10,      //comment1
                     $b=20,      //comment2
-                    $c=30) {
+                    $c=30
+                ) {
                 }',
             ],
             'must keep align comments (2)' => [
                 '<?php function xyz(
                     $a=10,  //comment1
                     $b=2000,//comment2
-                    $c=30) {
+                    $c=30
+                ) {
                 }',
             ],
             'multiline comments also must be ignored (I)' => [
-                '<?php function xyz(
+                <<<'EXPECTED'
+<?php function xyz(
```

Comment:
as I stated here: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2440#discussion_r116956645 , different indent is intended

Sure, it's not needed for every single test case, but for some of them it was important, even crucial, as it was testing reported bugs.
Let us not investigating "was it important in given case" for every test case you reindent. 
That's why golden rule with utests stated - add new tests, but don't modify existing ones.